packages/next-polyfill-module/src/index.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-extend-native */
+
+// Contains polyfills for methods missing after browser version(s):
+// Edge 16, Firefox 60, Chrome 61, Safari 10.1
+
+/**
+ * Available in:
+ * Edge: never
+ * Firefox: 61
+ * Chrome: 66
+ * Safari: 12
+ *
+ * https://caniuse.com/mdn-javascript_builtins_string_trimstart
+ * https://caniuse.com/mdn-javascript_builtins_string_trimend
+ */
+if (!('trimStart' in String.prototype)) {
+  String.prototype.trimStart = String.prototype.trimLeft
+}
+if (!('trimEnd' in String.prototype)) {
+  String.prototype.trimEnd = String.prototype.trimRight
+}
+
+/**
+ * Available in:
+ * Edge: never
+ * Firefox: 63
+ * Chrome: 70
+ * Safari: 12.1
+ *
+ * https://caniuse.com/mdn-javascript_builtins_symbol_description
+ */
+if (!('description' in Symbol.prototype)) {
+  Object.defineProperty(Symbol.prototype, 'description', {
+    get: function get() {
+      return /\((.+)\)/.exec(this)[1]
+    },
+  })
+}
```

Comment:
@ijjk This polyfill doesn't work properly on Edge 44 (https://github.com/vercel/next.js/issues/18213) and Safari 11 (https://github.com/vercel/next.js/issues/17825). We can't upgrade to 9.5.4 to fix the security alert and also 10.0 today... Does Next.js need this polyfill internally? If not, could we remove it for now?

Reply:
Can you send a PR to fix it?

Reply:
Yes, I can open a PR to fix the issue on Edge 44 with a small change
```js
Object.defineProperty(Symbol.prototype, 'description', {
  get: function get() {
    const result = /\((.+)\)/.exec(this)[1];
    return result ? result[1] : undefined;
  },
})
```
However, this still doesn't handle the other issue with Safari 11 (https://github.com/vercel/next.js/issues/17825). Core-js [`es.symbol.description`](https://github.com/zloirock/core-js/blob/master/packages/core-js/modules/es.symbol.description.js) polyfill seems to be more complete (handling all browser) but I'm not sure if we want to bring all that code into Next.js. I would suggest that we remove this problematic polyfill from Next.js. Whoever needs it can polyfill it manually.

@Timer WDYT?

Reply:
Core-js does not polyfill symbol.description, it polyfills Symbols themselves (in their entirety). This works:

```js
if (!('description' in Symbol.prototype)) {
  Object.defineProperty(Symbol.prototype, 'description', {
    get: function get() {
      return /\((.+)\)/.exec(this.toString())[1]
    },
  })
}
```